.travis.yml

```diff
@@ -1,11 +1,17 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-
+matrix:
+    include:
+        - os: linux
+          python: 2.6
+          cache: pip
```

Comment:
Let's enable this for the others in the interest of being consistent.

Reply:
I'm fine with that, very possible some not to daty versions still circulating for quite a while...